wasp-app-runner/src/smtp.ts

```diff
@@ -0,0 +1,24 @@
+import { exitCode } from "process";
+import { spawnWithLog } from "./process.js";
+import { log } from "./logging.js";
+
+export async function startLocalSmtpServer(): Promise<void> {
+  spawnWithLog({
+    name: "smtp-server",
+    cmd: "docker",
+    args: [
+      "run",
+      "--rm",
+      "-p",
+      "1080:1080",
+      "-p",
+      "1025:1025",
+      "marlonb/mailcrab:latest",
+    ],
+  }).then(({ exitCode }) => {
+    if (exitCode !== 0) {
+      log("smtp-server", "error", `SMTP server exited with code ${exitCode}`);
```

Comment:
This is a nice example of localized logs and their prefixes (I talked about that in the other PR).

Usually, I'd recommend all `process.exit` calls to be at the top level (i.e., bubble up the error and exit at the top). But I think we can let it slide in `wasp-app-runner`.